manifests/init.pp

```diff
@@ -9,15 +10,15 @@
   $charon_logging_conf = $strongswan::params::charon_logging_conf,
 ) inherits strongswan::params {
 
-
+  include apt
```

Comment:
I don't think we want to `include apt`.  Not least, this will break the module for all Redhat/CentOS users.  If you need to configure `apt update`, you should do this in your profile class.